<launch>

    <arg name="marker1Id"         default="1"/>
    <arg name="marker2Id"         default="3"/>
    <arg name="markerSize"        default="0.12"/>    <!-- in m -->
    <arg name="dct_normalization" default="False" />
    <arg name="dct_filter_size"   default="2" />
    <arg name="marker1_frame"     default="marker_przod" />
    <arg name="marker2_frame"     default="marker_tyl" />


    <node pkg="aruco_ros" type="double" name="aruco_simple">    
        <remap from="/camera_info" to="/follow/raspicam_node/camera_info" />
        <remap from="/image" to="/follow/raspicam_node/image" />
        <param name="image_is_rectified" value="True"/>
        <param name="marker_size" value="$(arg markerSize)"/>
        <param name="marker_id1" value="$(arg marker1Id)"/>
        <param name="marker_id2" value="$(arg marker2Id)"/>
        <param name="normalizeImage" value="$(arg dct_normalization)"/>
        <param name="dct_components_to_remove" value="$(arg dct_filter_size)"/>
        <param name="parent_name" value="stereo_gazebo_camera_optical_frame"/>
        <param name="child_name1" value="$(arg marker1_frame)" />
        <param name="child_name2" value="$(arg marker2_frame)" />
    </node>

</launch>
